src/pages/chooseTimeRestrictions.tsx

```diff
@@ -55,8 +55,10 @@ const ChooseTimeRestrictions = ({
                         dayCounters={dayCounters}
                     />
                     <label id="save-time-restriction-label" className="govuk-label" htmlFor="save-time-restriction">
```

Comment:
This htmlFor isn't actually pointing at anything, I would update it to be `time-restrictions-name` and then remove the `aria-describedby` on line 69 so the field is properly labelled